Makefile

```diff
@@ -24,14 +31,12 @@ else
   IMAGE_BUILD_CMD = docker build
 endif
 
-.PHONY: depend
```

Comment:
wouldn't `DOCKER_CMD` above need to be env `GO111MODULE` and `GOFLAGS` aware?

Reply:
ping @spangenberg

Reply:
Thanks, should be fixed now.